docs/api.rst

```diff
@@ -330,6 +330,31 @@ Generates Facebook login URL to request access token and permissions.
     fb_login_url = graph.auth_url(app_id, canvas_url, perms)
     print(fb_login_url)
 
+get_code_from_token
+^^^^^^^^^^^^^^^^^^^
+https://developers.facebook.com/docs/facebook-login/access-tokens/expiration-and-extension/
+
+Exchanges an existing long livd user access token on the server side for a
+code that can be redeemed for a long lived user token on the client side.
+
+**Parameters**
+
+* ``app_id`` - ``integer`` Facebook application id that is requesting a code.
+* ``app_secret`` - ``string`` Facebook application secret that is attempting
+  to get a code from the access token.
+* ``redirect_uri`` - ``string`` Return URL after successful authentication,
+  usually parses returned Facebook response for authorisation request.
+
```

Comment:
The "Parameters" section uses the same grammar as the `auth_url` method's documentation, but that section is wrong. Check any other method for better examples.